lang/java/tools/src/main/java/org/apache/avro/tool/IdlTool.java

```diff
@@ -38,28 +39,73 @@ public int run(InputStream in, PrintStream out, PrintStream err,
 
     PrintStream parseOut = out;
 
-    if (args.size() > 2 ||
-        (args.size() == 1 && (args.get(0).equals("--help") ||
-                              args.get(0).equals("-help")))) {
-      err.println("Usage: idl [in] [out]");
+    boolean incorrectArgsOrHelpRequested = false;
```

Comment:
Sry for the late reaction.

Yeah was thinking about commons, but did no want to add the dependency. I will try to update the pull request in the coming days.